plugin/trino-hive/src/main/java/io/trino/plugin/hive/s3/TrinoS3FileSystem.java

```diff
@@ -169,6 +171,7 @@
 import static java.util.concurrent.Executors.newCachedThreadPool;
 import static java.util.concurrent.TimeUnit.SECONDS;
 import static java.util.stream.Collectors.toList;
+import static org.apache.commons.lang3.StringUtils.isNotEmpty;
```

Comment:
avoid

Reply:
Prefer JDK and Guava. The two have generally consistent API rules, especially around null-friendliness.
Apache Commons has a different null-friendliness philosophy, and it should be avoided in the project.

Reply:
I replaced the check with `String.isBlank`